frontend/src/metabase/css/core/base.css

```diff
@@ -9,6 +9,17 @@ html {
   width: 100%;
 }
 
+@media print and (orientation: portrait) {
```

Comment:
Previously the styles above with `width: 100%` took precedence. Make it straightforward that we want to override the width for printing by placing overrides directly below the base styles.